openpype/hosts/blender/plugins/publish/extract_blend.py

```diff
@@ -23,24 +24,39 @@ def process(self, instance):
         # Perform extraction
         self.log.info("Performing extraction..")
 
+        plugin.deselect_all()
+
         data_blocks = set()
 
         for obj in instance:
             data_blocks.add(obj)
+            # Skip and non Object type like Collections.
+            if not isinstance(obj, bpy.types.Object):
+                continue
```

Comment:
If continuing, the `add` shouldn't be processed.
```suggestion
            # Skip and non Object type like Collections.
            if not isinstance(obj, bpy.types.Object):
                continue
            data_blocks.add(obj)
```

Reply:
Actually I want to add all instance data_blocks but juste skip for non object type to prevent invalid attribute error with obj.type because Collection hasn't this property. I use a "not condition" with continue to spare increments. I change this with simple condition.